AopCaching/readme.txt

```diff
@@ -0,0 +1,6 @@
+﻿-------------------------------------------
+PubComp.Caching.AopCaching
+-------------------------------------------
+
+Please make sure after this NuGet is installed, that the PstSharp NuGet is installed as well explicitly

```

Comment:
typo - PstSharp (missing o)

Reply:
I would rephrase that - if you are using the new style csproj or PackageReference tags in old csproj files, make sure you also install the PostSharp NuGet package (version 6.0.x)